python/setup.py

```diff
@@ -206,4 +222,10 @@ def get_package_data_files():
     os.remove("MANIFEST.in")
     for path in LIB_LIST:
         _, libname = os.path.split(path)
-        os.remove("tvm/%s" % libname)
+        path_to_be_removed = f"tvm/{libname}"
```

Comment:
nit: you use os.path.join everywhere else but not here (could be a follow-up)

Reply:
I'll send a few minor refactoring items in a separate PR - thanks!